### PR TITLE
Lint assessment controller

### PR DESCRIPTION
**Pull request by victorhuangwq**
## Description

Runs rubocop on assessment controller

## Motivation and Context

Part of effort to lint autolab codebase, to add rubocop

## How Has This Been Tested?

- Ran rails s, 
- tested submitting assessment, tried downloading handouts and viewing writeup

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

File: app/controllers/assessments_controller.rb [hunk 1]
```diff
@@ -67,11 +67,15 @@ class AssessmentsController < ApplicationController
   def index
     @is_instructor = @cud.has_auth_level? :instructor
     announcements_tmp = Announcement.where("start_date < :now AND end_date > :now",
-                                           now: Time.now)
+                                           now: Time.current)
```
Comment (victorhuangwq) [inline app/controllers/assessments_controller.rb:70]: Time.current is recommended by rubocop, and 
https://stackoverflow.com/questions/12748626/when-should-datetime-now-utc-vs-time-current-utc-be-used-in-rails

File: app/controllers/assessments_controller.rb [hunk 2]
```diff
@@ -13,16 +13,16 @@ class AssessmentsController < ApplicationController
     redirect_to("/home/error_404")
   end
 
-  autolab_require Rails.root.join("app", "controllers", "assessment", "handin.rb")
+  autolab_require Rails.root.join("app/controllers/assessment/handin.rb")
```
Comment (ashleyzhang) [inline app/controllers/assessments_controller.rb:16]: Is there a reason that putting the string with "/" directly is better?

**Review by ashleyzhang — COMMENTED** [review]
Just a quick question but everything else looks good!

**Review by ashleyzhang — APPROVED** [review]
LGTM!

**Pull request merged**